.circleci/config.yml

```diff
@@ -1,92 +1,87 @@
-version: 2
-jobs:
-  deploy_api:
-    docker:
-      -
+jobs: 
+  deploy_api: 
```

Comment:
Do we need 3 separate deployment jobs? Could we do it with 1 that deploys all three sections? Would use less containers on CI, of which we only have 4.